{"labels": [{"name": "tag", "value": "login"}, {"name": "suite", "value": "test_login_pytest"}, {"name": "subSuite", "value": "TestLoginP"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "host", "value": "G50-75M"}, {"name": "thread", "value": "15668-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_login_pytest"}], "historyId": "930813980c1d67c94f4cedd1773e7594", "fullName": "TestCases.test_login_pytest.TestLoginP#test_login_0_failed_by_wrong_datas[data5]", "uuid": "cba9bd82-c4ae-4d24-bce0-cc9169581909", "start": 1561889728375, "name": "test_login_0_failed_by_wrong_datas[data5]", "status": "passed", "parameters": [{"name": "data", "value": "{'user': '186847aa553', 'password': 'python', 'check': '请输入正确的手机号'}"}], "stop": 1561889729120}